src/components/sdk/MeetingControls/ContentShareControl.tsx

```diff
@@ -29,21 +30,51 @@ const ContentShareControl: React.FC<Props> = ({
     toggleContentShare,
     togglePauseContentShare,
   } = useContentShareControls();
+  const ref = useRef<HTMLInputElement>(null);
 
-  const dropdownOptions: PopOverItemProps[] = [
+  const dropdownOptionsShare: PopOverItemProps[] = [
     {
       children: <span>{paused ? unpauseLabel : pauseLabel}</span>,
       onClick: togglePauseContentShare,
     },
   ];
 
+  const fileUploadOnChangeHandler = () => {
```

Comment:
When I was debugging your WIP branch last week, I noticed that sometimes it would take two clicks to open the file menu. Did you run into this at all or did it get fixed?

Reply:
Yep, it is fixed. The click event handler was registered on the label element, which did not occupy the entire button area. Now it is working fine.

Reply:
Nice!